learning_journal_basic/templates/.layout.jinja2

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
Oh wait nevermind. I'm guessing that this came with the scaffold and you didn't want to make any mistakes with accessing it. You should delete this file along with `.mytemplate.jinja2`